doc/content/examples/example4.md

```diff
@@ -34,4 +31,70 @@ layer effectively simulates the interface between two elastic layers (see [fig:d
+
+- `initial_shear_modulus = '2.71e10'` specifies the small strain shear modulus (elastic property) of
+ the soil and is equivalent to the shear modulus of the concrete block. It should be noted that if we were to model an embedded foundation the shear modulus of the thin layer must match that of the surrounding soil strata.
+
+- `poissons_ratio = '0.15'` specifies the Poisson's ratio for the thin layer elements (same as concrete). The linear elastic properties are matched with concrete in order to ensure uniform scaling of stresses during failure.
+
+- `block = 1002` specifies that the material will be assigned to a mesh volume with block id = 1002.
+
+- `initial_stress = 'initial_ymid 0 0 0 initial_ymid 0 0 0 initial_zzmid` see [Getting Started](manuals/user/index.md). This is not needed if pressure dependency is set to true or when the initial stresses are unknown.
+
+- `density = '8792'` specifies the density of the thin layer which is the same as concrete.
+
+- `p_ref = '904918'` is the reference pressure at which the backbone curve is defined. The reference pressure is obtained by performing a gravity analysis and switching off the inertia kernel for the first few time steps. This helps us in avoiding unrealistic displacement/acceleration response.
+
+The documentation for computation of the finite elastic stresses and strains can be found in [modules/tensor_mechanics/index.md]
+
+!media media/examples/unittestcyclic.jpg
+      style=width:60%;margin-left:200px;float:center;
+      id=fig:deformedcontact
+      caption=Deformed shape of the numerical model.
+
+The boundary conditions and the supporting functions are given in the BCs and Functions block respectively.
+
+!listing examples/ex04/ex04b/2BlockFriction_Isoilunit.i
+ start=BC
+ end=Functions
+
+where;
+
+- `type = PresetBC` is used to declare fixed boundary condition for the soil block, where the displacements in the chosen boundaries in the x, y and z direction are assigned with the value 0.
+
+- `type = PresetDisplacement` This is used to define the displacement boundary condition on a specific boundary when inertial effects are taken into account. The PresetDisplacement takes the boundary id and the value of the displacement (which can be either expressed as constant value or as a function) as input.
+
+- `function = loading_bc'` specifies the displacement loading function to be input through the Functions block. The loading_bc in this problem is a sawtooth curve with the loading being applied after the first few time steps in order to stabilize gravity effects. The displacement and acceleration response output of the thin layer element must be verified with the input to check for discrepancies. It is advisable to use a displacement-time history with incremental amplitude to tackle sudden spikes in acceleration/velocity response due to convergence issues arising out of inconsistent timestep size input.
```

Comment:
Please replace "The displacement and acceleration response output of the thin layer element must be verified with the input to check for discrepancies." with something like "It is good practice to confirm that the displacement and acceleration output at the boundary where the displacement/acceleration is prescribed match the input displacement/acceleration time history."

Reply:
"It is advisable to use a displacement-time history with incremental amplitude to tackle sudden spikes in acceleration/velocity response due to convergence issues arising out of inconsistent timestep size input." is confusing. I am assuming this is for the sine input that you used for some other problem. I think the two points you are trying to convey here are:
1. When using PresetDisplacement boundary condition, the initial velocity and acceleration are assumed to be zero. While modeling a function with non-zero initial velocity such as a Asin(wt), it is better to linearly ramp up the amplitude of the sine wave, i.e., At sin(wt), over the first second or so and then maintain a constant amplitude. This ensures that the default initial conditions for velocity and acceleration are satisfied by the displacement time history. 
2. The sampling rate of the displacement time history should be smaller than the default time step that would be used in the simulation, i.e., if dt is 0.01, it is better to sample the displacement time history at 0.001 s. Mastodon automatically cuts the timestep by half if convergence is not achieved at a particular timestep. During these instances, if the displacement time history is only sampled at the default dt, the prescribed displacement at half of dt is obtained using linear interpolation which may result in artificial jumps in velocity/acceleration.